From: Tony Jones <tonyj@suse.de>
Subject: remove perfmon syscall_base attribute
Patch-mainline: never, see description in perfmon2.patch

Syscalls were removed, delete the old sysfs attribute that reported the 
syscall base.

Signed-off-by: Tony Jones <tonyj@suse.de>

---
---
 arch/ia64/include/asm/perfmon_kern.h |    5 -----
 perfmon/perfmon_sysfs.c              |   10 ----------
 2 files changed, 15 deletions(-)

--- a/arch/ia64/include/asm/perfmon_kern.h
+++ b/arch/ia64/include/asm/perfmon_kern.h
@@ -331,11 +331,6 @@ static inline void pfm_release_dbregs(st
 
 #define pfm_use_dbregs(_t)     __pfm_use_dbregs(_t)
 
-static inline int pfm_arch_get_base_syscall(void)
-{
-	return __NR_pfm_create_context;
-}
-
 struct pfm_arch_pmu_info {
 	unsigned long mask_pmcs[PFM_PMC_BV]; /* modify on when masking */
 };
--- a/perfmon/perfmon_sysfs.c
+++ b/perfmon/perfmon_sysfs.c
@@ -96,8 +96,6 @@ static struct kobj_type pfm_fmt_ktype =
 
 static ssize_t pfm_controls_show(struct kobject *kobj, struct kobj_attribute *attr, char *buf)
 {
-	int base;
-
 	if (is_attr_name(attr, "version"))
 		return snprintf(buf, PAGE_SIZE, "%u.%u\n",  PFM_VERSION_MAJ, PFM_VERSION_MIN);
 
@@ -115,12 +113,6 @@ static ssize_t pfm_controls_show(struct
 
 	if (is_attr_name(attr, "arg_mem_max"))
 		return snprintf(buf, PAGE_SIZE, "%zu\n", pfm_controls.arg_mem_max);
-
-	if (is_attr_name(attr, "syscall")) {
-		base = pfm_arch_get_base_syscall();
-		return snprintf(buf, PAGE_SIZE, "%d\n",  base);
-	}
-
 	if (is_attr_name(attr, "sys_sessions_count"))
 		return pfm_sysfs_res_show(buf, PAGE_SIZE, 1);
 
@@ -192,7 +184,6 @@ skip:
  */
 static PFM_RO_ATTR(version, pfm_controls_show);
 static PFM_RO_ATTR(task_sessions_count, pfm_controls_show);
-static PFM_RO_ATTR(syscall, pfm_controls_show);
 static PFM_RO_ATTR(sys_sessions_count, pfm_controls_show);
 static PFM_RO_ATTR(smpl_buffer_mem_cur, pfm_controls_show);
 
@@ -206,7 +197,6 @@ static PFM_RW_ATTR(reset_stats, pfm_cont
 
 static struct attribute *pfm_kernel_attrs[] = {
 	&attr_version.attr,
-	&attr_syscall.attr,
 	&attr_task_sessions_count.attr,
 	&attr_sys_sessions_count.attr,
 	&attr_smpl_buffer_mem_cur.attr,
